stylelint.config.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
   ],
   rules: {
     'max-nesting-depth': 3,
+    'no-extra-semicolons': true,
```

Comment:
This will be non-passive, can we update it to a warning until the next major version bump?

Reply:
Good point! https://github.com/cerner/stylelint-config-terra/pull/31/commits/f543f74e335ea2681ce7769f8f3ebf164e05c0ba

Reply:
Did we log an issue to fail this on the next MVB?

Reply:
https://github.com/cerner/stylelint-config-terra/issues/32